aten/src/ATen/core/type.cpp

```diff
@@ -1389,11 +1412,28 @@ void ClassType::checkForwardHookSchema(
 }
 
 torch::jit::Function* ClassType::findMethod(const std::string& name) const {
+  // if this is overloaded, there are multiple methods with the
+  // same name. Since this method is expected to work correctly
+  // only for normal methods, we just return nullptr
+  if (auto overloaded_methods = findOverloadedMethod(name)) {
+    if (overloaded_methods.value().size() == 1) {
+      return getMangledOverloadedMethod(overloaded_methods.value()[0]);
```

Comment:
There will be **at least** one overload for everything registered through custom class. This is because we don't know if the method will be overloaded or not so we just assume everything is overloaded. (custom_class.h)

Reply:
I see. Could we have a TODO to optimize this? Something like
1) assume method is not overloaded
2) move it to overloaded list when another method with same name registers

Reply:
done